packages/block-library/CHANGELOG.md

```diff
@@ -9,7 +9,6 @@
 ### New Feature
 
 -   Made it possible to import individual blocks ([#42258](https://github.com/WordPress/gutenberg/pull/42258)). Check [README](./README.md#loading-individual-blocks) for more information.
--   Paragraph block: You can now drop files/blocks/HTML on an empty Paragraph block to transform it into relevant blocks ([#42722](https://github.com/WordPress/gutenberg/pull/42722)).
```

Comment:
It accidentally went into the wrong release changelog while it hasn't been released yet. I think it's okay to remove it?

Reply:
If it's possible to move it to the right place that'd be good. It's probably not super important though, I don't know how many people look at the changelog for a package like block-library.

Reply:
Yeah, I moved the changelog to `block-editor`.